config/deployments/env_specific.toml

```diff
@@ -0,0 +1,211 @@
+# For explanantion of each config, please refer to the `config/config.example.toml` file
+
+[analytics.clickhouse]
+username = "clickhouse_username"     # Clickhouse username
+password = "clickhouse_password"     # Clickhouse password (optional)
+host = "http://localhost:8123"       # Clickhouse host in http(s)://<URL>:<PORT> format
+database_name = "clickhouse_db_name" # Clickhouse database name
+
+# Analytics configuration.
+[analytics]
+source = "sqlx" # The Analytics source/strategy to be used
+
+[analytics.sqlx]
+username = "db_user"      # Analytics DB Username
+password = "db_pass"      # Analytics DB Password
+host = "localhost"        # Analytics DB Host
+port = 5432               # Analytics DB Port
+dbname = "hyperswitch_db" # Name of Database
+pool_size = 5             # Number of connections to keep open
+connection_timeout = 10   # Timeout for database connection in seconds
+queue_strategy = "Fifo"   # Add the queue strategy used by the database bb8 client
+
+[api_keys]
+# Base64-encoded (KMS encrypted) ciphertext of the API key hashing key
+kms_encrypted_hash_key = ""
+
+[applepay_decrypt_keys]
+apple_pay_ppc = "APPLE_PAY_PAYMENT_PROCESSING_CERTIFICATE"         # Payment Processing Certificate provided by Apple Pay (https://developer.apple.com/) Certificates, Identifiers & Profiles > Apple Pay Payment Processing Certificate
+apple_pay_ppc_key = "APPLE_PAY_PAYMENT_PROCESSING_CERTIFICATE_KEY" # Private key generated by Elliptic-curve prime256v1 curve. You can use `openssl ecparam -out private.key -name prime256v1 -genkey` to generate the private key
+apple_pay_merchant_cert = "APPLE_PAY_MERCHNAT_CERTIFICATE"         # Merchant Certificate provided by Apple Pay (https://developer.apple.com/) Certificates, Identifiers & Profiles > Apple Pay Merchant Identity Certificate
+apple_pay_merchant_cert_key = "APPLE_PAY_MERCHNAT_CERTIFICATE_KEY" # Private key generated by RSA:2048 algorithm. Refer Hyperswitch Docs (https://docs.hyperswitch.io/hyperswitch-cloud/payment-methods-setup/wallets/apple-pay/ios-application/) to generate the private key
+
+[applepay_merchant_configs]
+common_merchant_identifier = "apple_pay_common_merchant_identifier"                        # This can be obtained by decrypting the apple_pay_ppc_key
+merchant_cert = "apple_pay_merchant_certificate"
+merchant_cert_key = "apple_pay_merchant_certificate_key"
+applepay_endpoint = "https://apple-pay-gateway.apple.com/paymentservices/registerMerchant"
+
+[connector_onboarding.paypal]
+client_id = "paypal_client_id"
+client_secret = "paypal_client_secret"
+enabled = true                         # boolean
```

Comment:
Let's keep this the first one in the section, so that users are easily aware that it's either enabled or disabled.